index.js

```diff
@@ -2258,24 +2267,60 @@ Jimp.prototype.print = function (font, x, y, text, maxWidth, cb) {
         return throwError.call(this, "text must be a string", cb);
     if ("number" != typeof maxWidth)
         return throwError.call(this, "maxWidth must be a number", cb);
+    if ("number" != typeof maxWidth)
+        return throwError.call(this, "alignment must be a number", cb);
```

Comment:
Probably forgot to change variable name in this line.  It's same as in the block above and judging from error message it should have been `alignment`